go/vt/vtcombo/tablet_map.go

```diff
@@ -33,6 +31,7 @@ import (
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/mysqlctl"
 	"vitess.io/vitess/go/vt/mysqlctl/tmutils"
+	"vitess.io/vitess/go/vt/proto/vschema"
```

Comment:
for consistency, this should probably be `vschemapb` (and with the other `/proto/` imports below) (sorry i missed this one last time!)

Reply:
`vschemapb` it is